spec/services/authorisation_spec.rb

```diff
@@ -110,4 +110,20 @@
       end
     end
   end
-end
+
+  describe '#many?' do
+    subject do
+      described_class.new(
+        organisation_id: '939eac36-0777-48c2-9c2c-b87c948a9ee0',
+        user_id: '161d1f6a-44f1-4a1a-940d-d1088c439da7'
+      )
+    end
+
+    before(:each) do
+      stub_sign_in_with_multiple_organisations
+    end
+    it 'should check if the user is a member of multiple organisations' do
```

Comment:
We've only got 1 test at the moment to cover this functionality.  I have no cofidence that:

- The `user_id` above is being passed into the URL
- The method doesn't always return true

Can we get some triangulation going?

Reply:
Added some test to check for sign in with a single organisation and also for different user id in 414ede9